documentation/Makefile

```diff
@@ -0,0 +1,29 @@
+SRC = $(wildcard *.md)
+
+DISCOURSE_MD = $(addprefix discourse/, $(SRC))
+HTML_MD = $(addprefix html/, $(SRC))
+HTML = $(HTML_MD:.md=.html)
+
+discourse/%.md: %.md
+	@mkdir --parents discourse
```

Comment:
gitignore entries for these directories would be nice

Reply:
Good point! Added.